lib/tasks/check_content_consistency.rake

```diff
@@ -1,32 +1,31 @@
-namespace :check_content_consistency do
-  def check_content(content_id, locale, ignore_recent)
-    checker = ContentConsistencyChecker.new(content_id, locale, ignore_recent)
-    errors = checker.call
+require "content_consistency_checker"
 
-    if errors.any?
-      puts "#{content_id} #{locale} 😱"
-      puts errors
-    end
+def report_errors(errors, content_store)
+  Airbrake.notify(
+    "Inconsistent #{content_store} documents",
+    parameters: {
+      errors: errors,
+    }
+  )
 
-    errors.none?
+  errors.each do |base_path, item_errors|
+    puts "#{base_path} 😱"
+    puts item_errors
   end
+end
 
-  desc "Check documents for consistency with the router-api and content-store"
-  task :one, [:content_id, :locale] => [:environment] do |_, args|
-    content_id = args[:content_id]
-    locale = args[:locale] || "en"
-    check_content(content_id, locale, false)
-  end
+desc "Check all the documents for consistency with the router-api and content-store"
```

Comment:
is this actually just content-store? also this implies that it does the work independently whereas it's actually given the data